NitroxPatcher/Patches/Dynamic/Stalker_CheckLoseTooth_Patch.cs

```diff
@@ -8,22 +9,48 @@ public sealed partial class Stalker_CheckLoseTooth_Patch : NitroxPatch, IDynamic
 {
     private static readonly MethodInfo TARGET_METHOD = Reflect.Method((Stalker t) => t.CheckLoseTooth(default(GameObject)));
 
-    //GetComponent<HardnessMixin> was returning null for everything instead of a HardnessMixin with a hardness value. Since this component
-    //isn't used for anything else than the stalker teeth drop, we hard-code the values and bingo.
+    // HardnessMixin seems to be a bit buggy (ie: undefined values for some scraps, which is a vanilla bug), so we'll just hard-code the values for now.
+    // Note that HardnessMixin is only used by Stalkers
     public static bool Prefix(Stalker __instance, GameObject target)
```

Comment:
Is there a reason we can't trust the logic from the original method that does these checks for how the loose teeth works and we have to do the spawning etc for ourselves?

Reply:
Would need to transpile the LooseTooth method, because I need the gameObject to broadcast the item packet
But since I already have this patch that invalidate the whole original method, and LooseTooth is only use here. It can simplify a bit.

Reply:
Sounds sensible to me. Just thought I would question it as to the reasoning behind it. Creating a transpiler without need doesn't seem sensible in this case then